### PR TITLE
i3ipc v0.2

### PR DESCRIPTION
**Pull request by Armael**
(thanks to @pdonadeo who contributed all of the changes of this new release)

### CODE REVIEW

**Comment by mseri** [comment]
I think `ocaml >= 4.02.0` should be updated to `>= 4.03.0`:
```
#=== ERROR while compiling i3ipc.0.2 ==========================================#
# context     2.0.3 | linux/x86_64 | ocaml-base-compiler.4.02.3 | file:///home/travis/build/ocaml/opam-repository
# path        ~/.opam/ocaml-base-compiler.4.02.3/.opam-switch/build/i3ipc.0.2
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -p i3ipc -j 1
# exit-code   1
# env-file    ~/.opam/log/i3ipc-21147-0a5907.env
# output-file ~/.opam/log/i3ipc-21147-0a5907.out
### output ###
#          ppx src/i3ipc.pp.ml (exit 1)
# (cd _build/default && .ppx/0c26f60c5b4b9d4a67fc98ed4e677950/ppx.exe --cookie 'library-name="i3ipc"' -o src/i3ipc.pp.ml --impl src/i3ipc.ml --dump-ast)
# File "src/i3ipc.ml", line 27, characters 16-25:
# Cannot locate deriver of_yojson
```

File: packages/i3ipc/i3ipc.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+bug-reports: "https://github.com/Armael/ocaml-i3ipc/issues"
+license: "MIT"
+tags: ["i3" "ipc" "window-manager"]
+dev-repo: "git+https://github.com/Armael/ocaml-i3ipc"
+doc: "https://armael.github.io/ocaml-i3ipc/0.2/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.02.0"}
```
Comment (mseri) [inline packages/i3ipc/i3ipc.0.2/opam]:
```suggestion
  "ocaml" {>= "4.03.0"}
```

Comment (Armael) [reply packages/i3ipc/i3ipc.0.2/opam]: Indeed, thanks.

**Comment by mseri** [comment]
Thanks

**Pull request merged**